gf-webapp/src/logic/rooms/roomsPage/roomBlock/roomItem/information/Information.tsx

```diff
@@ -1,9 +1,12 @@
 import {useDictionary} from "src/logic/DictionaryContext/useDictionary";
 import styles from "src/logic/rooms/roomsPage/roomBlock/roomItem/RoomItem.module.scss";
+import {RoomType} from "src/model/Room/RoomType";
```

Comment:
Please move this line on second position and styles on third

Reply:
Done

Reply:
Good